Winapp2.ini

```diff
@@ -6217,13 +6223,8 @@ Section=Games
 Detect=HKCU\Software\Epic Games\EpicGamesLauncher
 Default=False
 FileKey1=%LocalAppData%\EpicGamesLauncher\Saved\Crashes|*.*|REMOVESELF
-FileKey2=%LocalAppData%\EpicGamesLauncher\Saved\Logs|*.log
-
-[Epic Games Launcher Cache *]
-Section=Games
-Detect=HKCU\Software\Epic Games\EpicGamesLauncher
-Default=False
-FileKey1=%LocalAppData%\EpicGamesLauncher\Saved\webcache|*.*|REMOVESELF
+FileKey2=%LocalAppData%\EpicGamesLauncher\Saved\Logs|*.*|REMOVESELF
+FileKey3=%LocalAppData%\EpicGamesLauncher\Saved\webcache|*.*|REMOVESELF
```

Comment:
We decided to keep most cache entries in separate entries like this, because cache files are not technically junk to some users and they help speed up software.

Reply:
Alright, I've separated the cache entries for:
- Epic Games Launcher
- Fortnite